extension/test/test-logger.js

```diff
@@ -27,7 +33,12 @@ var verifyAsyncOutput = function (test, logger, testFunc) {
     }
     finally {
       istream.close();
-      message = JSON.parse(message);
+      try {
+        message = JSON.parse(message);
```

Comment:
I don't know that much of the addon SDK. @KWierso could you shed some light into here?